mgmtsystem_nonconformity/models/mgmtsystem_nonconformity_abstract.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+
+from odoo import api, fields, models
+from odoo.tools.misc import frozendict
+
+
+class MgmtsystemNonconformityAbstract(models.AbstractModel):
+    # TODO: Remove this on 17.0 and move everything on mail.thread
+    _name = "mgmtsystem.nonconformity.abstract"
+    _description = "Nonconformity Abstract"
+
+    non_conformity_ids = fields.One2many(
+        "mgmtsystem.nonconformity",
+        inverse_name="res_id",
+        domain=lambda r: [("res_model", "=", r._name)],
+        readonly=True,
```

Comment:
If a field is marked as readonly=True on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the readonly attribute on the views instead.
```suggestion
```